packages/core/addon/components/table-renderer-vertical-collection.js

```diff
@@ -100,6 +105,15 @@ export default Component.extend({
     );
 
     get(this, 'tableHeadersDomElement').addEventListener(WHEEL_EVENT, e => this._headerWheelSync(e));
+
+    //chrome better behavior
+    if (typeof window.ResizeObserver !== 'undefined') {
```

Comment:
should we still use the service in case it's supported?

Reply:
I'm gonna look into a more comprehensive solution with the polyfill and yeah it won't use resize service when I'm done with it.

Reply:
oops missed the do not review label